docusaurus.config.js

```diff
@@ -164,6 +166,15 @@ const config = {
               },
             ],
           },
+          {
+            title: 'Contact Us',
+            items: [
+              {
+                label: 'Email',
+                href: '#contact-email',
```

Comment:
This link won't work from any page except the landing page because the script checks for `location.pathname === '/'`. Let's do:

```suggestion
                href: '/#contact-email',
```

Reply:
Let's also add an email link to the end of the navbar:

```js
          {
            href: "/#contact-email",
            'aria-label': "Contact Email",
            html: '<i class="fa-regular fa-envelope"></i>',
            position: "right",
            className: "navbar-icon",
          }
```